ansible/playbook/roles/cluster/tasks/main.yml

```diff
@@ -48,3 +48,15 @@
 - name: Show cluster up output
   debug:
     var: clusterupout.stdout_lines
+
+- block:
+  - debug:
+      msg: "Enabling ASB"
+
+  - name: Enable additional features such as automation service broker
+    command: "oc cluster add --base-dir={{ cluster_base_dir }} {{ item }}"
+    register: clusterenable
+    with_items:
+    - service-catalog
```

Comment:
Can we avoid to hard code such values and to let the user to pass this parameter then to the playbook `-e enable="service-catalog,-openshift-automation-broker"` ?

Reply:
Having `-...` doesn't make sense, we can only add things with `oc cluster add`, not remove them.

So yes we can have a flag named `enable` which we will use to add things.
If that is what we want to do, then I will close this PR and open a new one